lms/djangoapps/ccx/migrations/0004_change_ccx_coach_to_staff.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import logging
+
+from django.db import migrations
+# We're doing something awful here, but it's necessary for the greater good:
+from django.contrib.auth.models import User
```

Comment:
Thanks for clarifying. Agreed, and I believe the only piece of the User model actually being relied on here is it's primary key (both with CCX coach mapping and user roles). I doubt there will be a conflicting migration that messes with the User key (at least not for a very very long time).